mlflow/lightgbm.py

```diff
@@ -379,6 +380,22 @@ def log_feature_importance_plot(features, importance, importance_type):
         else:
             kwargs["callbacks"] = [callback]
 
+        # We set free_raw_data to false on the Dataset object
+        # so that we can access the original data later.
+        train_data = args[1] if len(args) > 1 else kwargs["train_data"]
+
+        input_example = None
+        try:
+            if isinstance(train_data.data, str):
+                raise Exception("The input data was of type string.")
+
+            SAMPLE_ROWS = 5
+            input_example = deepcopy(train_data.data[:SAMPLE_ROWS])
+        except Exception as e:  # pylint: disable=broad-except
+            input_example = None
```

Comment:
Nit: I think this line is unnecessary; if the assignment to `input_example` via `deepcopy()` fails, `input_example will still be `None` (as defined above). Alternatively, instead of removing this instance, you could also remove the one on line 387.